custom/icds/messaging/indicators.py

```diff
@@ -403,7 +403,7 @@ def get_messages(self, language_code=None):
         num_weigh = self.get_value_from_fixture(self.weighed_fixture, 'open_weighed')
         num_weigh_avail = self.get_value_from_fixture(self.weighed_fixture, 'open_count')
         num_days_open = int(self.get_value_from_fixture(self.days_open_fixture, 'awc_opened_count'))
-        num_awc_locations = len(self.days_open_fixture.findall('./rows/row[@is_total_row="False"]'))
+        num_awc_locations = len(self.awc_locations)
```

Comment:
using `len(self.awc_locations)` as the denominator here is better for the same reason - AWCs could drop out of the actual report